### PR TITLE
Fixed an issue dropping with out placeholder

### LINKED ISSUE #456: On fast drops the drop event fires before placeholder is created and component breaks
### supporting information:
- I use https://lukasoppermann.github.io/html5sortable/index.html

### How can the issue be reproduce the problem?
Just drag as fast as you can many elements from a list to another list like in the picture. Usually after 10 tries you should be able to break it.
Real users reported this, probably on slower computers it happens even more, no need to stress the app.

evidence
more-evidence

### PR DESCRIPTION
**Pull request by alsundukov**
https://github.com/lukasoppermann/html5sortable/issues/456
Fixes #456

Similar changes were made in the commit bc046961 Avery Lemons, but all this was in the docs directory

### CODE REVIEW

File: src/html5sortable.ts [hunk 1]
```diff
@@ -475,11 +475,16 @@ export default function sortable (sortableElements, options: configuration|objec
...
-      // remove placeholder from dom
-      visiblePlaceholder.remove()
-
+      if (visiblePlaceholder) {
+        // attach element after placeholder
+        after(visiblePlaceholder, dragging)
+        // remove placeholder from dom
+        visiblePlaceholder.remove()
+      } else {
+        // set the dropped value to 'true' to delete copied dragging at the time of 'dragend'
```
Comment (lukasoppermann) [inline src/html5sortable.ts]: @alsundukov you write set to `true` but you are setting it to `false`, can you explain this?

File: docs/html5sortable.js [hunk 2]
```diff
@@ -1114,11 +1114,16 @@ var sortable = (function () {
                   .filter(function (placeholder) { return placeholder instanceof HTMLElement; })
                   // only elements in DOM
                   .filter(isInDom)[0];
-              // attach element after placeholder
-              if (visiblePlaceholder && visiblePlaceholder.nextElementSibling != null) {
-                insertAfter(visiblePlaceholder, dragging);
-                // remove placeholder from dom
-                visiblePlaceholder.remove();
+              if (visiblePlaceholder) {
```
Comment (lukasoppermann) [inline docs/html5sortable.js]: Can you please remove this file from the PR, it will be auto-generated on build. Thanks

**Comment by lukasoppermann** [comment]
Hey @alsundukov thanks for the PR, I added a question.

Can you also please rebase the PR?

Would it be possible to include this in the tests, so that we do note have a decrease in coverage? Thank you.

**Comment by alsundukov** [comment]
Hey @lukasoppermann, I did it

**Comment by lukasoppermann** [comment]
Hey @alsundukov great, thank you.

Can you please also reply / act on the two comments I added in the review? Thanks

**Comment by alsundukov** [comment]
Sure, but I don't see your questions, sorry

**Comment by lukasoppermann** [comment]
Sorry, it was somehow still pending, you should see it now.

File: src/html5sortable.ts (hunk 1 above)
Comment (alsundukov) [reply src/html5sortable.ts]: ops, this is a typo in the comment

File: docs/html5sortable.js (hunk 2 above)
Comment (alsundukov) [reply docs/html5sortable.js]: Yes, of course

**Comment by alsundukov** [comment]
Done

**Review by lukasoppermann — APPROVED** [review]

**Comment by lukasoppermann** [comment]
Great, thank you.

**Pull request merged**